src/batou/secrets/encryption.py

```diff
@@ -258,13 +263,43 @@ def get_identities():
                 "~/.ssh/id_dsa",
             ]
         # filter on existing files
-        identities = [
+        paths = [
             os.path.expanduser(x)
             for x in identities
             if os.path.exists(os.path.expanduser(x))
         ]
+
         if debug:
-            print(f"Found identities: {identities}", file=sys.stderr)
+            print(f"Found identities: {paths}", file=sys.stderr)
+
+        def load(id_path):
+            with open(id_path, "rb") as f:
+                key_content = f.read()
+            try:
+                priv_key = serialization.load_ssh_private_key(key_content, None)
+            except ValueError:
```

Comment:
This actually should be 

```suggestion
            except (ValueError, TypeError):
```

I ran into an issue where `batou secrets edit staging` failed with the error message thrown inside `cryptography`
```
Key is password-protected, but password was not provided.
```
that is a `TypeError`